sandcats/public/privacy.html

```diff
@@ -0,0 +1,250 @@
+        } else {
+          document.title = "Sandstorm Sandcats: Terms of Service";
+        }
+      }
+      function showPrivacy() {
+        document.body.className = "privacy";
+        if (history.replaceState) {
+          history.replaceState({}, "Sandstorm Sandcats: Privacy Policy", "/privacy.html");
+        } else {
+          document.title = "Sandstorm Sandcats: Privacy Policy";
+        }
+      }
+    </script>
+    <style type="text/css">
+body {
+  font-family: sans-serif;
+  background-color: #eee;
+  padding: 0;
+  margin: 0;
+}
+
+section > * {
+  margin: 1em auto;
+  max-width: 640px;
+  box-sizing: border-box;
+}
+
+section > blockquote {
+  padding: 0 32px;
+  background-color: #eee;
+  overflow: hidden;
+}
+
+h1 {
+  font-size: 25pt;
+}
+
+h3 {
+  font-size: inherit;
+}
+
+#terms, #privacy {
+  position: absolute;
+  background-color: white;
+  margin: 64px 0 0;
+  border-top: 1px solid #888;
+  width: 100%;
+}
+
+#terms::before, #privacy::before {
+  display: block;
+  position: absolute;
+  top: -32px;
+  height: 15px;
+  border: 1px solid #888;
+  border-bottom: none;
+  background-color: white;
+  padding: 8px 0;
+  text-align: center;
+  width: 190px;
+}
+
+#terms::before {
+  content: "Terms of Service";
+  left: calc(50% - 320px);
+}
+
+#privacy::before {
+  content: "Privacy Policy";
+  left: calc(50% - 120px);
+}
+
+body.terms #terms, body.privacy #privacy {
+  z-index: 1;
+}
+
+body.privacy #terms > *, body.terms #privacy > * {
+  display: none;
+}
+
+@media screen and (max-width: 650px) {
+  section > * {
+    width: 100%;
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+  #terms::before, #privacy::before {
+    width: calc(50% - 16px);
+  }
+  #terms::before { left: 8px; }
+  #privacy::before { right: 8px; left: auto; }
+}
+
+    </style>
+  </head>
+  <body class="privacy">
+    <section id="overview">
+<h1>Overview: Sandstorm Sandcats</h1>
+
+<p>Sandstorm is a platform for server-oriented and web-based applications. Sandstorm the platform is open source; anyone can run Sandstorm on their own personal server hardware. When you run Sandstorm on your own hardware, you can do anything you want with it: Sandstorm the platform is licensed under the Apache 2.0 open source license which places no restrictions on usage.</p>
+
+<p>Sandstorm the company (Sandstorm Development Group, Inc.) also offers a DNS service called Sandcats. Sandcats makes it easy to obtain a hostname and TLS certificate for your personal Sandstorm server, as a subdomain of sandcats.io. Sandstorm owns the domain sandcats.io but delegates a subdomain, such as "example.sandcats.io", to you for free. The purpose of the service is to make it easy for you and your friends to easily find your Sandstorm server on the network and ensure that they are communicating with it securely. Basically, it acts like a phone book. The Sandcats service never sees any of the content of your server nor any of the traffic to your server.</p>
```

Comment:
Consider rephrasing this "The company that produces the Sandstorm platform (Sandstorm Development Group, Inc.)" rather than the colloquial "Sandstorm the company" which doesn't really parse and could be confusing if you've not heard someone use that phrasing before aloud.